applications/StructuralMechanicsApplication/tests/beam_test/semi_rigid_linear_3D2NBeamCr_test_parameters.json

```diff
@@ -0,0 +1,123 @@
+        "residual_relative_tolerance"        : 1e-12,
+        "residual_absolute_tolerance"        : 1e-12,
+        "max_iteration"                      : 100,
+        "rotation_dofs"                      : true
+    },
+    "processes" : {
+    "constraints_process_list" : [{
+        "python_module" : "assign_vector_variable_process",
+        "kratos_module" : "KratosMultiphysics",
+        "help"                  : "This process fixes the selected components of a given vector variable",
+        "process_name"          : "AssignVectorVariableProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "Structure.DISPLACEMENT_dirichletXYZ",
+            "variable_name"   : "DISPLACEMENT",
+            "constrained"	  : [true,true,true],
+            "value"           : [0.0,0.0,0.0]
+        }
+    },{
+        "python_module" : "assign_vector_variable_process",
+        "kratos_module" : "KratosMultiphysics",
+        "help"                  : "This process fixes the selected components of a given vector variable",
+        "process_name"          : "AssignVectorVariableProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "Structure.DISPLACEMENT_dirichletYZ",
+            "variable_name"   : "DISPLACEMENT",
+            "constrained"	  : [false,false,false],
+            "value"           : [0.0,0.0,0.0]
+        }
+    },{
+        "python_module" : "assign_vector_variable_process",
+        "kratos_module" : "KratosMultiphysics",
+        "help"                  : "This process fixes the selected components of a given vector variable",
+        "process_name"          : "AssignVectorVariableProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "Structure.ROTATION_dirrot",
+            "variable_name"   : "ROTATION",
+            "constrained"	  : [true,true,true],
+            "value"           : [0.0,0.0,0.0]
+        }
+    },{
+        "python_module" : "assign_scalar_variable_to_entities_process",
+        "kratos_module" : "KratosMultiphysics",
+        "process_name"          : "AssignScalarVariableToEntitiesProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "Structure.Parts_semi_rigid_hinge",
+            "variable_name"   : "ROTATIONAL_STIFFNESS_AXIS_2",
+            "value"           : 0.0,
+            "entities"        : ["nodes"]
+        }
+    },{
+        "python_module" : "assign_scalar_variable_to_entities_process",
+        "kratos_module" : "KratosMultiphysics",
+        "process_name"          : "AssignScalarVariableToEntitiesProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "Structure.Parts_semi_rigid_hinge",
+            "variable_name"   : "ROTATIONAL_STIFFNESS_AXIS_3",
```

Comment:
FYI you can directly write this in the mdpa, up to you
https://github.com/KratosMultiphysics/Kratos/blob/e2ec33153ec8b7f6c6a61eb41ce2ee04621603ed/kratos/tests/auxiliar_files_for_python_unittest/mdpa_files/test_model_part_io_read.mdpa#L73-L78

Reply:
ah thanks